<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1045</width>
    <height>711</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QDockWidget" name="ramp_widget">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>321</width>
      <height>161</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>300</width>
      <height>149</height>
     </size>
    </property>
    <property name="floating">
     <bool>false</bool>
    </property>
    <property name="windowTitle">
     <string>Ramp</string>
    </property>
    <widget class="QWidget" name="dockWidgetContents_6">
     <layout class="QGridLayout" name="gridLayout_6">
      <item row="0" column="0">
       <layout class="QFormLayout" name="formLayout_2">
        <item row="0" column="0">
         <widget class="QLabel" name="label_8">
          <property name="text">
           <string>Frequency (Hz)</string>
          </property>
         </widget>
        </item>
        <item row="0" column="1">
         <widget class="QDoubleSpinBox" name="ramp_frequency_DoubleSpinBox">
          <property name="maximum">
           <double>200.000000000000000</double>
          </property>
         </widget>
        </item>
        <item row="1" column="0">
         <widget class="QLabel" name="label_10">
          <property name="text">
           <string>Amplitude (V)</string>
          </property>
         </widget>
        </item>
        <item row="1" column="1">
         <widget class="QDoubleSpinBox" name="ramp_amplitude_DoubleSpinBox">
          <property name="decimals">
           <number>3</number>
          </property>
          <property name="minimum">
           <double>0.000000000000000</double>
          </property>
          <property name="maximum">
           <double>1.860000000000000</double>
          </property>
          <property name="singleStep">
           <double>0.010000000000000</double>
          </property>
         </widget>
        </item>
        <item row="2" column="0">
         <widget class="QPushButton" name="StartRamp_PushButton">
          <property name="text">
           <string>Start ramp</string>
          </property>
         </widget>
        </item>
        <item row="2" column="1">
         <widget class="QPushButton" name="StopRamp_PushButton">
          <property name="text">
           <string>Stop ramp</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </widget>
   <widget class="QDockWidget" name="cavityposition">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>160</y>
      <width>300</width>
      <height>536</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="minimumSize">
     <size>
      <width>300</width>
      <height>300</height>
     </size>
    </property>
    <property name="windowTitle">
     <string>Cavity Position</string>
    </property>
    <widget class="QWidget" name="dockWidgetContents_7">
     <layout class="QFormLayout" name="formLayout_3">
      <item row="1" column="1">
       <widget class="QDoubleSpinBox" name="doubleSpinBox_2">
        <property name="decimals">
         <number>3</number>
        </property>
        <property name="maximum">
         <double>20.000000000000000</double>
        </property>
        <property name="singleStep">
         <double>0.001000000000000</double>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="label_11">
        <property name="text">
         <string>Target position (um)</string>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <widget class="QPushButton" name="GOpushButton">
        <property name="text">
         <string>Go</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1045</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <widget class="QDockWidget" name="transmissionmap">
   <property name="sizePolicy">
    <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="minimumSize">
    <size>
     <width>500</width>
     <height>407</height>
    </size>
   </property>
   <property name="windowTitle">
    <string>Transmission Map</string>
   </property>
   <attribute name="dockWidgetArea">
    <number>1</number>
   </attribute>
   <widget class="QWidget" name="dockWidgetContents">
    <layout class="QHBoxLayout" name="horizontalLayout_3">
     <item>
      <widget class="PlotWidget" name="transmission_map_PlotWidget"/>
     </item>
     <item>
      <layout class="QGridLayout" name="gridLayout_5">
       <item row="0" column="0">
        <widget class="ScienDSpinBox" name="transmission_map_cb_max_DoubleSpinBox">
         <property name="maximumSize">
          <size>
           <width>100</width>
           <height>16777215</height>
          </size>
         </property>
         <property name="alignment">
          <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
         </property>
         <property name="suffix">
          <string>c/s</string>
         </property>
         <property name="minimum">
          <double>-10.000000000000000</double>
         </property>
         <property name="maximum">
          <double>1000000000.000000000000000</double>
         </property>
         <property name="value">
          <double>100000.000000000000000</double>
         </property>
        </widget>
       </item>
       <item row="5" column="0">
        <widget class="QRadioButton" name="transmission_map_cb_centiles_RadioButton">
         <property name="text">
          <string>Per&amp;centiles</string>
         </property>
         <property name="checked">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item row="4" column="0">
        <widget class="ScienDSpinBox" name="transmission_map_cb_min_DoubleSpinBox">
         <property name="maximumSize">
          <size>
           <width>100</width>
           <height>16777215</height>
          </size>
         </property>
         <property name="alignment">
          <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
         </property>
         <property name="suffix">
          <string>c/s</string>
         </property>
         <property name="minimum">
          <double>-10.000000000000000</double>
         </property>
         <property name="maximum">
          <double>1000000000.000000000000000</double>
         </property>
         <property name="value">
          <double>1000.000000000000000</double>
         </property>
        </widget>
       </item>
       <item row="2" column="0">
        <widget class="PlotWidget" name="transmission_map_cb_PlotWidget">
         <property name="sizePolicy">
          <sizepolicy hsizetype="MinimumExpanding" vsizetype="MinimumExpanding">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="maximumSize">
          <size>
           <width>80</width>
           <height>16777215</height>
          </size>
         </property>
        </widget>
       </item>
       <item row="1" column="0">
        <widget class="ScienDSpinBox" name="transmission_map_cb_high_percentile_DoubleSpinBox">
         <property name="maximumSize">
          <size>
           <width>100</width>
           <height>16777215</height>
          </size>
         </property>
         <property name="alignment">
          <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
         </property>
         <property name="suffix">
          <string>%</string>
         </property>
         <property name="minimum">
          <double>0.000000000000000</double>
         </property>
         <property name="maximum">
          <double>100.000000000000000</double>
         </property>
         <property name="value">
          <double>100.000000000000000</double>
         </property>
        </widget>
       </item>
       <item row="3" column="0">
        <widget class="ScienDSpinBox" name="transmission_map_cb_low_percentile_DoubleSpinBox">
         <property name="maximumSize">
          <size>
           <width>100</width>
           <height>16777215</height>
          </size>
         </property>
         <property name="alignment">
          <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
         </property>
         <property name="suffix">
          <string>%</string>
         </property>
         <property name="minimum">
          <double>0.000000000000000</double>
         </property>
         <property name="maximum">
          <double>100.000000000000000</double>
         </property>
         <property name="value">
          <double>1.000000000000000</double>
         </property>
        </widget>
       </item>
       <item row="6" column="0">
        <widget class="QRadioButton" name="transmission_map_cb_manual_RadioButton">
         <property name="text">
          <string>Man&amp;ual</string>
         </property>
        </widget>
       </item>
       <item row="7" column="0">
        <widget class="QCheckBox" name="normalize_checkBox">
         <property name="text">
          <string>Normalize_data</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QDockWidget" name="dockWidget_2">
   <property name="sizePolicy">
    <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="windowTitle">
    <string>Spectrum</string>
   </property>
   <attribute name="dockWidgetArea">
    <number>1</number>
   </attribute>
   <widget class="QWidget" name="dockWidgetContents_2">
    <layout class="QGridLayout" name="gridLayout_2">
     <item row="0" column="0">
      <widget class="PlotWidget" name="spectrum_PlotWidget"/>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QDockWidget" name="sweep_parameters">
   <property name="windowTitle">
    <string>Sweep Parameters</string>
   </property>
   <attribute name="dockWidgetArea">
    <number>2</number>
   </attribute>
   <widget class="QWidget" name="dockWidgetContents_3">
    <layout class="QGridLayout" name="gridLayout_3">
     <item row="6" column="0">
      <widget class="QPushButton" name="continue_pushButton">
       <property name="text">
        <string>continue</string>
       </property>
      </widget>
     </item>
     <item row="0" column="0">
      <widget class="QLabel" name="label">
       <property name="text">
        <string>Start position (um)</string>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string>Stop position (um)</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QDoubleSpinBox" name="start_position_doubleSpinBox"/>
     </item>
     <item row="5" column="0">
      <widget class="QPushButton" name="start_pushButton">
       <property name="text">
        <string>start</string>
       </property>
      </widget>
     </item>
     <item row="3" column="1">
      <widget class="QDoubleSpinBox" name="scan_speed_doubleSpinBox">
       <property name="decimals">
        <number>3</number>
       </property>
       <property name="value">
        <double>0.020000000000000</double>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QDoubleSpinBox" name="stop_position_doubleSpinBox">
       <property name="value">
        <double>2.000000000000000</double>
       </property>
      </widget>
     </item>
     <item row="5" column="1">
      <widget class="QPushButton" name="stop_pushButton">
       <property name="text">
        <string>stop</string>
       </property>
      </widget>
     </item>
     <item row="3" column="0">
      <widget class="QLabel" name="label_3">
       <property name="text">
        <string>Scan frequency (Hz)</string>
       </property>
      </widget>
     </item>
     <item row="4" column="1">
      <widget class="QDoubleSpinBox" name="number_of_steps_doubleSpinBox">
       <property name="decimals">
        <number>0</number>
       </property>
       <property name="minimum">
        <double>1.000000000000000</double>
       </property>
       <property name="maximum">
        <double>1000.000000000000000</double>
       </property>
       <property name="value">
        <double>10.000000000000000</double>
       </property>
      </widget>
     </item>
     <item row="4" column="0">
      <widget class="QLabel" name="label_6">
       <property name="text">
        <string>Number of steps</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QDockWidget" name="sectrometercontrol">
   <property name="windowTitle">
    <string>Spectrometer  Control</string>
   </property>
   <attribute name="dockWidgetArea">
    <number>2</number>
   </attribute>
   <widget class="QWidget" name="dockWidgetContents_4">
    <layout class="QGridLayout" name="gridLayout_4">
     <item row="4" column="0">
      <widget class="QLabel" name="label_5">
       <property name="text">
        <string>Exposure time</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QDoubleSpinBox" name="set_temperature_doubleSpinBox">
       <property name="decimals">
        <number>0</number>
       </property>
       <property name="minimum">
        <double>-100.000000000000000</double>
       </property>
       <property name="maximum">
        <double>100.000000000000000</double>
       </property>
      </widget>
     </item>
     <item row="4" column="1">
      <widget class="QDoubleSpinBox" name="exposure_time_doubleSpinBox">
       <property name="decimals">
        <number>3</number>
       </property>
       <property name="maximum">
        <double>5000.000000000000000</double>
       </property>
       <property name="value">
        <double>1.000000000000000</double>
       </property>
      </widget>
     </item>
     <item row="7" column="0">
      <widget class="QPushButton" name="take_spectrum_pushButton">
       <property name="text">
        <string>Take Spectrum</string>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QPushButton" name="set_temperature_pushButton">
       <property name="text">
        <string>Set Temperature</string>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="QDoubleSpinBox" name="get_temperature_doubleSpinBox">
       <property name="decimals">
        <number>0</number>
       </property>
       <property name="minimum">
        <double>-100.000000000000000</double>
       </property>
      </widget>
     </item>
     <item row="3" column="1">
      <widget class="QDoubleSpinBox" name="averages_doubleSpinBox">
       <property name="decimals">
        <number>0</number>
       </property>
       <property name="value">
        <double>1.000000000000000</double>
       </property>
      </widget>
     </item>
     <item row="3" column="0">
      <widget class="QLabel" name="label_4">
       <property name="text">
        <string>Number of accumulations</string>
       </property>
      </widget>
     </item>
     <item row="2" column="0">
      <widget class="QPushButton" name="get_temperature_pushButton">
       <property name="text">
        <string>Get Current Temperature</string>
       </property>
      </widget>
     </item>
     <item row="5" column="1">
      <widget class="QDoubleSpinBox" name="cycle_time_doubleSpinBox">
       <property name="decimals">
        <number>3</number>
       </property>
       <property name="maximum">
        <double>5000.000000000000000</double>
       </property>
       <property name="value">
        <double>1.000000000000000</double>
       </property>
      </widget>
     </item>
     <item row="5" column="0">
      <widget class="QLabel" name="label_7">
       <property name="text">
        <string>Cycle time</string>
       </property>
      </widget>
     </item>
     <item row="0" column="0">
      <widget class="QPushButton" name="pushButton_cooler_on">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Minimum" vsizetype="Preferred">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="text">
        <string>Cooler on</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QPushButton" name="pushButton_cooler_off">
       <property name="text">
        <string>Cooler off</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QDockWidget" name="dockWidget">
   <property name="windowTitle">
    <string>Pzt</string>
   </property>
   <attribute name="dockWidgetArea">
    <number>1</number>
   </attribute>
   <widget class="QWidget" name="dockWidgetContents_5">
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="PlotWidget" name="pzt_PlotWidget"/>
     </item>
    </layout>
   </widget>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>PlotWidget</class>
   <extends>QGraphicsView</extends>
   <header>pyqtgraph</header>
  </customwidget>
  <customwidget>
   <class>ScienDSpinBox</class>
   <extends>QDoubleSpinBox</extends>
   <header>qtwidgets.scientific_spinbox.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
